src/core/or/command.c

```diff
@@ -512,6 +514,11 @@ command_process_relay_cell(cell_t *cell, channel_t *chan)
     /* Count the payload bytes only. We don't care about cell headers */
     ocirc->n_read_circ_bw = tor_add_u32_nowrap(ocirc->n_read_circ_bw,
                                                CELL_PAYLOAD_SIZE);
+
+    /* Stash the original delivered and overhead values, so we can inform the
+     * control port about dropped cells immediately after processing below. */
+    orig_delivered_bw = ocirc->n_delivered_read_circ_bw;
```

Comment:
What parts of the code can modify these values and we are keeping the originals? Perhaps we can put it in the comment.

Reply:
Clarified comment in 02b5a6c67f2d3287ffd2d5ca773ff3534d1c774c